versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -12,6 +12,72 @@ import ContactLink from "@site/src/components/ContactLink";
 import Tabs from "@theme/Tabs";
 import TabItem from "@theme/TabItem";
 
+## `2.23.0` -> `2.24.0`
+
+## ImageAdapters without multiple sources
+
+In the version we extended the `ImageAdapters` feature allow an adapter to
+change how sources are genereated for the srcSet, to allow this, we had to
+[make changes to the `makeImagesProps`](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/merge_requests/2037)
+If you have overriden this file, you will need to update it to match the new
+implementation.
+
+```js title="src/web/theme/components/atoms/Image/makeImagesProps.js"
+// add-next-line
+import imageAdapters from "./adapters";
+
+const makeImgProps = (baseUrl, src, format, bg, cover, fluid) => {
+  // [..]
+
+  // add-start
+  const sourcesExtensions = imageAdapters.getSupportedExtensions(
+    src,
+    bg === "transparent"
+  );
+  const defaultExtension = imageAdapters.getDefaultExtension(src);
+  // add-end
+
+  return {
+    // remove-next-line
+    src: makeImgSrc(src, format, sizes[0], bg, extension, cover, baseUrl),
+    // add-next-line
+    src: makeImgSrc(src, format, sizes[0], bg, defaultFormat, cover, baseUrl),
+    sources: sourcesExtensions.map((extension) => ({
+      // remove-next-line
+      type: `image/${extension}`,
+      // add-next-line
+      ...(extension ? { type: `image/${extension}` } : {}),
+      srcSet: makeImageSrcset(
+        baseUrl,
+        src,
+        format,
+        sizes,
+        bg,
+        extension,
+        cover,
+        fluid
+      ),
+    })),
+  };
+};
+```
+
+We also added two optional methods `getSupportedExtensions` and
+`getDefaultExtension` to allow an adapter to change how sources are genereated
+for the srcSet, see the
```

Comment:
It could be simplified to prevent rewriting the same sentence than before
```suggestion
We also added two optional methods `getSupportedExtensions` and
`getDefaultExtension` to adapters
```